### PR TITLE
Salesforce README.md

### PR DESCRIPTION
**Pull request by dat-a-man**
# Tell us what you do here

- [ ] implementing verified source (please link a relevant issue labelled as `verified source`)
- [ ] fixing a bug (please link a relevant bug report)
- [Y ] improving, documenting, customizing existing source (please link an issue or describe below)
- [ ] anything else (please link an issue or describe below)

# Relevant issue

issue #

# More PR info
Added Salesforce README.md

### CODE REVIEW

File: sources/salesforce/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| Opportunity | merge | represents a sales opportunity for a specific account or contact |
+| OpportunityLineItem | merge | represents individual line items or products associated with an Opportunity |
+| OpportunityContactRole | merge | represents the association between an Opportunity and a Contact |
+| Account | merge | individual or organization that interacts with your business |
+| CampaignMember | merge | association between a Contact or Lead and a Campaign |
+| Task | merge | used to track and manage various activities and tasks within the Salesforce platform |
+| Event | merge | used to track and manage calendar-based events, such as meetings, appointments calls, or any other time-specific activities |
+
+To learn more about the write depositions i.e. merge, append and replace please refer to our official [documentation.](https://dlthub.com/docs/general-usage/incremental-loading)
+## Initialize the pipeline with Hubspot verified source
```
Comment (AstrakhantsevaAA) [inline sources/salesforce/README.md:31]: Hubspot

File: sources/salesforce/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    
+3. To make sure that everything is loaded as expected, use the command:
+    ```bash
+    dlt pipeline <pipeline_name> show
+    ```
+    
+    For example, the pipeline_name for the above pipeline is `salesforce`, you may also use any custom name instead.
+    
+
+    💡 To explore additional customizations for this pipeline, we recommend referring to the official dlt Salesforce verified documentation. It provides comprehensive information and guidance on how to further customize and tailor the pipeline to suit your specific needs. You can find the dlt Salesforce documentation in [Setup Guide: Salesforce.](https://dlthub.com/docs/dlt-ecosystem/verified-sources/salesforce)
```
Comment (AstrakhantsevaAA) [inline sources/salesforce/README.md:76]: it looks like a part of number 3. delete tabulation (select -> shift+tab)

File: sources/salesforce/README.md (hunk 1 above)
Comment (dat-a-man) [reply sources/salesforce/README.md]: DONE

File: sources/salesforce/README.md (hunk 2 above)
Comment (dat-a-man) [reply sources/salesforce/README.md]: done

**Review by AstrakhantsevaAA — APPROVED** [review]

**Pull request merged**